website/pages/blog/announcing-policies-in-cloudquery-hub.md

```diff
@@ -10,6 +10,8 @@ import { BlogHeader } from "../../components/BlogHeader"
 
 <BlogHeader/>
 
+**HCL policies were deprecated - see up-to-date policy documentation [here](https://www.cloudquery.io/docs/core-concepts/policies)**
```

Comment:
Can we use a Callout block here instead?

```
import { Callout } from 'nextra-theme-docs'
```

```
<Callout> 

HCL policies were deprecated - see up-to-date policy documentation [here](https://www.cloudquery.io/docs/core-concepts/policies)

</Callout>
```

Reply:
Hmmm, not sure if our blogs support `.mdx` (they're all `.md` right now).
I think making sure they support it is out-of-scope of this PR?

Reply:
They support `mdx` and there is even no need to rename it so it should just work

Reply:
:) !